docs/en/week07/07-3.md

```diff
@@ -0,0 +1,133 @@
+---
+lang-ref: ch.07-3
+title: Introduction to autoencoders
+lecturer: Alfredo Canziani
+authors: Vidit Bhargava, Monika Dagar
+date: 18 March 2021
+---
+# DALL-E: Creating Images from Text
+
+[![hackmd-github-sync-badge](https://hackmd.io/usp9PiUGRdGQzD93wE8Bsw/badge)](https://hackmd.io/usp9PiUGRdGQzD93wE8Bsw)
+
+A neural network that creates images from text captions. 
+![](https://i.imgur.com/FosonFM.jpg)
+
+
+Go to the [website](https://openai.com/blog/dall-e/) and play with the captions! 
+
+# Autoencoder
+Let's start with some definitions:
+
+## Definitions
+### Input
+
+$\boldsymbol{x}$: is observed during both training and testing 
+
+$\boldsymbol{y}$: is observed during training but not testing
+
+$\boldsymbol{z}$: is not observed (neither during training nor during testing).
+
+### Output
+$\boldsymbol{h}$: is computed from the input (hidden/internal)
+
+$\boldsymbol{\tilde{y}}$: is computed from the hidden (predicted $\boldsymbol{y}$, ~ means $circa$)
+
+Confused?
+Refer to the below figure to understand the use of different variables in different machine learning techniques.
+![](https://i.imgur.com/Y4CFlZc.png)
+
+## Autoencoder
+
+These kind of networks are used to learn the internal structure of an input and encode it in a hidden internal representation ($\boldsymbol{h}$) which expresses the input.
+
+We already learned how to train energy-based models, let's look at the below network:
+![](https://i.imgur.com/NAUmcJd.png)
+
+Here instead of computing the minimization of the energy $E$ with respect to $\boldsymbol{z}$, we use an encoder that approximates the minimization and provides a hidden representation $\boldsymbol{h}$ for a given $\boldsymbol{y}$.
+$$
+\boldsymbol{h} = Encoder(\boldsymbol{y})
```

Comment:
Use `\text{}` for unconventional function names. Here and everywhere else in this document.